### ISSUE TITLE
[bug]: WiFi on RaspberryPi - Country selection doesn't stick and continually re-prompts

### ISSUE THREAD

**Issue by gStart9** [Start9Labs/start-os#2545]
### Prerequisites

- [X] I have searched for [existing issues](https://github.com/start9labs/start-os/issues) that already report this problem.

### Server Hardware

Pi (4 and 8GB)

### StartOS Version

0.3.5.1

### Client OS

Linux

### Client OS Version

Debian 12

### Browser

Firefox

### Browser Version

120

### Current Behavior

In the web-UI on a Pi, under System > WiFi, even after successfully selecting a Country, the UI prompts the user to select it again:
![image](https://github.com/Start9Labs/start-os/assets/106188942/2d0b1303-2096-4d69-983e-3a01599ed232)
^Note in this screenshot that the country is successfully sent to the backend.

Even after selecting a Country AND successfully connecting to a wifi AP, the UI prompts the user to select their country again:
![image](https://github.com/Start9Labs/start-os/assets/106188942/f75c7799-c873-4904-90c6-08b452035f7e)

No real errors in the browser console to speak of:
![image](https://github.com/Start9Labs/start-os/assets/106188942/f1e2f645-1c7f-44bd-8a3d-2457a0f825ea)
Okin reported this error in the browser console:
> main.db4020b2e4a1127a.js:1 ecode: -32600error: code: -32600data: "missing field `country`"message: "Invalid Request"[[Prototype]]: Objectconstructor: ƒ Object()hasOwnProperty: ƒ hasOwnProperty()isPrototypeOf: ƒ isPrototypeOf()propertyIsEnumerable: ƒ propertyIsEnumerable()toLocaleString: ƒ toLocaleString()toString: ƒ ()valueOf: ƒ valueOf()__defineGetter__: ƒ __defineGetter__()__defineSetter__: ƒ __defineSetter__()__lookupGetter__: ƒ __lookupGetter__()__lookupSetter__: ƒ __lookupSetter__()__proto__: (...)get __proto__: ƒ __proto__()set __proto__: ƒ __proto__()message: "RPC ERROR: Invalid Request\n\nmissing field `country`"[[Prototype]]: Objectconstructor: class egetMessage: ƒ getMessage()[[Prototype]]: Object
(anonymous) @ main.db4020b2e4a1127a.js:1
> main.db4020b2e4a1127a.js:1 ecode: -32600error: {code: -32600, message: 'Invalid Request', data: 'missing field `country`'}message: "RPC ERROR: Invalid Request\n\nmissing field `country`"[[Prototype]]: Object
(anonymous) @ main.db4020b2e4a1127a.js:1

### Expected Behavior

Users expect for their country selection to stick.

### Steps to Reproduce

Boot up a pi on 0351
System > Wifi
Attempt to select your country

**Comment by gStart9** [comment]
https://github.com/Start9Labs/start-os/pull/2283/files
^this might be related.  I know we fixed this before with `crda` (and something else I don't quite recall, probably a command to set the region to 00?)
I noticed crda is missing from (what I presume is) the new version of that file:
https://github.com/Start9Labs/start-os/blob/v0.3.5.1/build/dpkg-deps/depends

**Comment by BigBossRabbit** [comment]
After speaking with George, we decided it best for me to add the findings from testing the WiFi connectivity for my My `Dell Inspiron 15 3000 with 8GB RAM & 1 TB HDD (StartOS v0.3.5.1)`:

- After restarting the Dell laptop 3 times & running through the WiFi process twice per restart, it has now connected to wifi. However, the Country has switched to `US -United States` but is working.
- The Dell however doesn't stay in the loop as is the experience with the pi.
- Once I disconnect the Ethernet cable on the Dell the WiFi only works if the country is US. So could it be that the other codes just don't get enforced & are NULL?
- I select Namibia, after a few tries it has it saved with the Green Tick but then after I disconnect the ethernet cable the server is disconnected from the internet.

What I did then was plug the ethernet cable back in & it still showed that the wifi had the green tick. So I restarted the server, with the ethernet still plugged in, went back to wifi & saw the green tick, then disconnected the ethernet again & this time it stayed on Wifi.

That was when I realised the country had switched to US like in the screenshot, even though I'm in Namibia.
<img width="1440" alt="Screenshot 2023-12-27 at 12 36 18 AM" src="https://github.com/Start9Labs/start-os/assets/87260669/327848bc-81a8-4c37-8386-96691a5530f2">
<img width="1440" alt="Screenshot 2023-12-27 at 12 37 45 AM" src="https://github.com/Start9Labs/start-os/assets/87260669/452cfe23-6e4e-45d8-8550-a75b950b1d22">
<img width="1439" alt="Screenshot 2023-12-27 at 12 36 59 AM" src="https://github.com/Start9Labs/start-os/assets/87260669/836ed33a-7dec-4629-a6a1-3d00ace990e6">
<img width="1439" alt="Screenshot 2023-12-27 at 12 15 14 AM" src="https://github.com/Start9Labs/start-os/assets/87260669/2e0a41df-baba-4e8e-b9ff-735b7f494a61">

**Comment by dr-bonez** [comment]
Looks like raspberry pi requires this specific version of the package: http://archive.raspberrypi.org/debian/pool/main/w/wireless-regdb/

**Comment by dr-bonez** [comment]
@BigBossRabbit are you on the `nonfree` iso?

**Comment by BigBossRabbit** [comment]
> @BigBossRabbit are you on the `nonfree` iso?

Yes, I believe so. I've been running the machine strictly on WiFi since this test was done.

**Comment by MattDHill** [comment]
@gStart9 @BigBossRabbit please test this once we have a 036 Ras Pi image.

**Comment by BigBossRabbit** [comment]
@MattDHill will definitely do. My RasPi is ready & waiting 🫡

**Comment by MattDHill** [comment]
@BigBossRabbit can you test this now please?

**Comment by BigBossRabbit** [comment]
Hi @MattDHill I'll gladly test but will only be able to test it once back in Namibia around the 27th or 28th of November. 

That is when I'll be next to a raspi again.

**Comment by MattDHill** [comment]
@BigBossRabbit did you manage to test this yet?

**Comment by BigBossRabbit** [comment]
I have not yet, but just got my old original Start9 Server back (the Raspi one I believe), so will update that & see if this still persists. I'll also try with my DIY Raspi machine. Just need to download the new images.

**Comment by BigBossRabbit** [comment]
Tested this out last night and the country selection still doesn't stick, with the box to choose a country reappearing continuously now when a selection has already been made. 

I will attempt to get screenshots of this later today and do a reflash to see if that might clear anything up.

**Comment by BigBossRabbit** [comment]
I was unfortunately not able to get a Screenshot before leaving for holiday, but will attempt to see if I can get the same error as I did before when back at the lab in about a week.

**Comment by BigBossRabbit** [comment]
<img width="1280" alt="Screenshot 2025-01-09 at 15 36 51" src="https://github.com/user-attachments/assets/c2f9d9aa-2e58-45ca-af2d-92a88d99e6b8" />
<img width="1280" alt="Screenshot 2025-01-09 at 15 37 15" src="https://github.com/user-attachments/assets/b445a0ca-0342-4fd4-85a0-db66798aa527" />
<img width="1279" alt="Screenshot 2025-01-09 at 15 38 28" src="https://github.com/user-attachments/assets/7a3f84ce-c4d9-454f-9535-4a5a61f1488b" />

I am still having the same issues after having done a fresh install of StartOS. Even though the final screenshot says that the connection has been successful, after waiting an hour and restarting the server, I still do not have a wifi connection enabled or even visible on the wifi tab @MattDHill

**Comment by MattDHill** [comment]
WiFI deprecated in 040, not planning to fix this.
